### DISCUSSION TITLE
should we ignore pnpm-store dir?

### DISCUSSION THREAD

**Discussion by saiqulhaq** [pnpm/pnpm#6936] [Q&A] [answered]
when I try to run this app on my local https://github.com/creatorsgarten/techcal.dev
I found there is .pnpm-store dir, is it ok to gitignore it?
if not ok, then I will close my PR https://github.com/creatorsgarten/techcal.dev/pull/108

**Comment by gluxon** [comment, marked as answer]
Hey @saiqulhaq — Your intuition to `.gitignore` this directory is correct.

If the project is `git clone`'ed to the root of a partition, `.pnpm-store` can end up in the root of the project.

From https://pnpm.io/npmrc#store-dir
> If there is no home on the disk, then the store is created at the root of the filesystem. For example, if installation is happening on a filesystem mounted at `/mnt`, then the store will be created at `/mnt/.pnpm-store`

In other words, what likely happened was that the "root of the filesystem" and your git clone were the same location. Others have seen this in the past and adding `.pnpm-store` to `.gitignore` is generally recommended since it isn't intended to be directly committed for normal projects.
